src/services/dashboards/dashboard-detail/lib/type.ts

```diff
@@ -2,3 +2,8 @@ import type { WIDGET_WIDTH_RANGE_LIST, WIDGET_WIDTH_FULL } from '@/services/dash
 
 // = 'SM' | 'MD' | 'LG' | 'XL' | 'FULL';
 export type WidgetSize = keyof typeof WIDGET_WIDTH_RANGE_LIST | typeof WIDGET_WIDTH_FULL;
+
+export interface WidgetThemeOption {
+    inherit: boolean;
+    inherit_count: number|undefined;
+}
```

Comment:
suggestion: I recommend to use `WidgetConfig` interface to reduce manage point :)

for example: 
```suggestion
export type WidgetThemeOption = WidgetConfig[theme]
```